### PR TITLE
0.8.1

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/examples/pull/284

cc @GuillaumeGomez, @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
:+1:

File: src/pollable_output_stream.rs [hunk 1]
```diff
@@ -155,11 +155,11 @@ pub struct OutputStreamAsyncWrite<T: IsA<PollableOutputStream>>(
 );
 
 impl<T: IsA<PollableOutputStream>> OutputStreamAsyncWrite<T> {
-    pub fn into_input_stream(self) -> T {
+    pub fn into_output_stream(self) -> T {
         self.0
     }
 
-    pub fn input_stream(&self) -> &T {
+    pub fn output_stream(&self) -> &T {
```
Comment (sdroege) [inline src/pollable_output_stream.rs]: This is an API change, and I'm aware of code out there using the old (broken) names. If anything we can deprecate the broken names here and add the new ones for this bugfix release (and that probably makes sense to do!).

Comment (EPashkin) [reply src/pollable_output_stream.rs]: Something like https://github.com/gtk-rs/gio/pull/279

**Comment by EPashkin** [comment]
I forgot to add one commit, added it now.
Also I will cherry pick #279 after it accepted

File: src/pollable_output_stream.rs (hunk 1 above)
Comment (EPashkin) [reply src/pollable_output_stream.rs]: Updated PR

**Comment by sdroege** [comment]
Thanks, looks good to me :) Will merge once the CI is green

**Comment by EPashkin** [comment]
CI passed

**Pull request merged**